packages/components/src/components/files/FileTree.tsx

```diff
@@ -246,11 +263,11 @@ export class FileTree extends PureComponent<FileTreeProps, FileTreeState> {
     };
 
     // Callback to parent with actual path of selected file
-    onFileSelect = (id: string, checked: boolean, isDirectory: boolean): void => {
+    onFileSelect = (id: string, checked: boolean, isDirectory: boolean, node: any): void => {
```

Comment:
I didn't notice this when I did the initial implementation, but looks like someone has already done a little work defining the types here in this issue.  If you are up for providing some types.  (Although looks like it's missing node.selected).

https://github.com/storybookjs/react-treebeard/issues/186#issuecomment-502162650

Reply:
I'm not going to take that on as part of this PR, but I'll add a TODO comment to the component.